api-lambda-ses/example-pattern.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "Amazon API Gateway REST API to AWS Lambda to Amazon SES",
+  "description": "This sample project demonstrates publishing an email through SES by invoking a Lambda function through a REST API.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "Create an Amazon API Gateway REST API that invokes a Lambda function to send email through SES.",
+    "text": ["This pattern deploys a Amazon API Gateway REST API with Lambda Function integration and SES"]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/blob/main/apigw-lambda-ses",
+      "templateURL": "serverless-patterns/apigw-lambda-ses",
+      "projectFolder": "apigw-lambda-ses",
+      "templateFile": "template.yml"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Publishing Emails in Amazon SES.",
+        "link": "https://docs.aws.amazon.com/ses/latest/dg/getting-started.html"
+      },
+      {
+        "text": "Creating and verifying identities in Amazon SES.",
+        "link": "https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html"
+      }
+    ]
+  },
+  "deploy": {
+    "text": [
+      "sam deploy --guided --parameter-overrides  SENDER=sender@email.com RECIPIENT=recipient@email.com"
+    ]
+  },
+  "testing": {
+    "text": [
+      "See the GitHub repo for detailed testing instructions."
+    ]
+  },
+  "cleanup": {
+    "text": [
+      "Delete the stack: <code>sam delete </code>."
+    ]
+  },
+  "authors": [
+    {
+      "name": "Himanshu Goyal",
+      "image": "https://media.licdn.com/dms/image/D4D03AQGfOBBJOHCFnw/profile-displayphoto-shrink_200_200/0/1674582869052?e=1704326400&v=beta&t=PMspUbSZaTe1IrbT_W6kzowHJXEKQLd-YIicHKTTRmM",
```

Comment:
The image isn't loading for me, I'm getting a 404 not found error.

Reply:
Updated the image URL.